src/station_cmd.cpp

```diff
@@ -89,25 +89,25 @@ bool IsHangar(TileIndex t)
 }
 
 /**
- * Look for a station around the given tile area.
+ * Look for a station owned by the given company around the given tile area.
  * @param ta the area to search over
- * @param closest_station the closest station found so far
+ * @param closest_station the closest owned station found so far
+ * @param owner the company whose stations to look for
  * @param st to 'return' the found station
  * @return Succeeded command (if zero or one station found) or failed command (for two or more stations found).
  */
 template <class T>
-CommandCost GetStationAround(TileArea ta, StationID closest_station, T **st)
+CommandCost GetStationAround(TileArea ta, StationID closest_station, OwnerByte owner, T **st)
```

Comment:
Just a tiny nit-pick: The `...Byte` types are usually only used for save/load. In normal functions, `Owner` is preferred, even if just for aesthetics.

Reply:
Changed this to CompanyID now, perhaps that's a bit more appropriate here. What do you think?